pkg/expression/context.go

```diff
@@ -96,16 +95,9 @@ func wrapEvalAssert(ctx EvalContext, fn builtinFunc) (ret *assertionEvalContext)
 }
 
 func checkEvalCtx(ctx EvalContext) {
-	loc := ctx.Location().String()
 	tc := ctx.TypeCtx()
-	tcLoc := tc.Location().String()
```

Comment:
We moved most asserts to `SessionEvalContext`. So we only need to check the `Expression.Location()` and `EvalContext.TypeCtx().Locaion()` are the same